recipes/delta-spark/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "delta-spark" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/delta-spark-{{ version }}.tar.gz
+    sha256: 8a22b75c21aafe0d5d05e38be012b932d48441465a19c49ac658136afdee0933
+  - url: https://raw.githubusercontent.com/delta-io/delta/v{{ version }}/version.sbt
+    sha256: 5ef30d70070bff442e8a09223e1432a204b2016e0b73b589781037c4fd45e039
+  - url: https://raw.githubusercontent.com/delta-io/delta/v{{ version }}/python/README.md
+    sha256: dc28ef3b3f0a1c2b3cbfd144b06d7c831762cc13cb04440792456f615149bb25
+    folder: python
```

Comment:
If most of the files needed are not included in the PyPI source, it is easier to use GitHub release.

In this case https://github.com/delta-io/delta/archive/refs/tags/v2.2.0.tar.gz and you can create the sha256 with `curl -sL https://github.com/username/reponame/archive/vX.X.X.tar.gz | openssl sha256`